buildscripts/CMakeLists.txt

```diff
@@ -37,16 +37,16 @@ set(MYGUI_VERSION 3.2.2)
 set(MYGUI_HASH SHA256=0a28d7ec8a47993cb68deb48b36331e28f12dd92580b709eaef21d599b67a78f)
 
 # https://github.com/ptitSeb/gl4es
-set(GL4ES_VERSION v1.1.2)
-set(GL4ES_HASH SHA256=0774a1d0500c3a5d4da2d8cf3054ce4780a53cd20fe607024e1ceb38823f195e)
+set(GL4ES_VERSION d8f996bf76196e43adaf01e266b36aba62310e67)
```

Comment:
Currently my policy is to stick to gl4es stable releases, otherwise it's too much working keeping it up to date.

Reply:
Newest have important changes, like fix for water when using shaders(shader rendered object had wrong reflection), or some texture fixes (noticed two bad things with gl4es 1.1.2 with lysol texture packs), also the most important one texture coordinates was passed badly to shaders with 1.1.2, it was the main issue why they didnt worked properly.